chsdi/models/vector/bak.py

```diff
@@ -10,30 +10,84 @@
 Base = bases['bak']
 
 
-class Isos(Base, Vector):
-    __tablename__ = 'isos'
-    __table_args__ = ({'autoload': False})
+class IsosBase:
     __template__ = 'templates/htmlpopup/isos.mako'
+    __table_args__ = ({'autoload': False, 'extend_existing': True})
+    id = Column('xtf_id', Unicode, primary_key=True)
+    kantone = Column('kantone', Unicode)
+    nummer = Column('id', Integer)
+    name = Column('name', Unicode)
+    siedlungskategorie = Column('siedlungskategorie', Unicode)
+    url = Column('url', Unicode)
+
+
+class IsosOrtsbild(Base, IsosBase, Vector):
+    __tablename__ = 'view_isos_ortsbild'
+    __bodId__ = 'ch.bak.bundesinventar-schuetzenswerte-ortsbilder'
+    __label__ = 'name'
+    __queryable_attributes__ = ['id', 'name']
+    __minscale__ = 50000
+    the_geom = Column(Geometry2D)
+
+
+class IsosOrtsbildPerimeter(Base, IsosBase, Vector):
+    __tablename__ = 'view_isos_ortsbild'
+    __bodId__ = 'ch.bak.bundesinventar-schuetzenswerte-ortsbilder'
+    __label__ = 'name'
+    __queryable_attributes__ = ['id', 'name']
+    __returnedGeometry__ = 'perimeter'
+    __maxscale__ = 50000
+    __minscale__ = 25000
+    perimeter = Column('perimeter', Geometry2D)
```

Comment:
Somehow `the_geom = Column('perimeter', Geometry2D)` throws an error even though there are some working examples.
Also this (polygon) geometry is not recognized as a clickable area to identify, only some point behind.

Reply:
I don't know - what happens when the attribute always is called the_geom?

the_geom = Column('perimeter', Geometry2D)

Reply:
As mentioned it throws an error upon deploying mf-chsdi3.

Reply:
did you try to make two views instead with spatial column the_geom?

Reply:
whats the error message?